python_modules/libraries/dagster-components/dagster_components_tests/code_locations/component_component_deps_via_scope/defs/script_depends_on_my_python_defs/component.yaml

```diff
@@ -0,0 +1,9 @@
+type: dagster_components.dagster.PipesSubprocessScriptCollectionComponent
+
+attributes:
+  scripts:
+    - path: script_one.py
+      assets:
+        - key: script_depends_on_my_python_defs
+          automation_condition: "{{ automation_condition.eager() }}"
+          deps: "{{ defs('component_component_deps_via_scope.defs.my_python_defs').get_all_asset_specs() }}"
```

Comment:
I feel like these module paths should be relative to the defs folder (e.g. defs('my_python_defs') ), as I don't think we should support loading defs from a module that's not in the same root directory, and if we don't support that then it feels pretty weird to force people to type all the my_project_name.defs stuff)

Reply:
updated